streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -313,7 +323,8 @@ public void shouldReturnTaskOffsetSumsMapForDecodedSubscription() {
             new SubscriptionInfo(MIN_VERSION_OFFSET_SUM_SUBSCRIPTION,
                                  LATEST_SUPPORTED_VERSION, UUID_1,
                                  "localhost:80",
-                                 TASK_OFFSET_SUMS)
+                                 TASK_OFFSET_SUMS,
+                                 IGNORED_UNIQUE_FIELD)
                 .encode());
         assertThat(info.taskOffsetSums(), is(TASK_OFFSET_SUMS));
     }
```

Comment:
I think we should also add a test to make sure that if you pass in a `uniqueField` to the SubscriptionInfo but the `usedVersion` is less than 8, that it does not actually encode this field.